crates/neon/src/context/mod.rs

```diff
@@ -551,20 +613,37 @@ pub trait Context<'a>: ContextInternal<'a> {
 }
 
 /// An execution context of module initialization.
-pub struct ModuleContext<'a> {
-    env: Env,
-    exports: Handle<'a, JsObject>,
+pub struct ModuleContext<'cx> {
+    cx: Cx<'cx>,
+    exports: Handle<'cx, JsObject>,
+}
+
+impl<'cx> Deref for ModuleContext<'cx> {
+    type Target = Cx<'cx>;
+
+    fn deref(&self) -> &Self::Target {
+        &self.cx
+    }
 }
 
-impl<'a> UnwindSafe for ModuleContext<'a> {}
+impl<'cx> DerefMut for ModuleContext<'cx> {
```

Comment:
Does this mean if you already have an `&mut ModuleContext` you'd have to do something like `&mut *cx` to pass it to a function that takes `Cx`?

Reply:
Nope! It can be passed directly and deref coercion will make it the right type. This is similar to how you can pass a `&String` to a function that takes a `&str`.

The only times you would need to be more explicit are if it's wrapped in another value (e.g., `Option<ModuleContext>` would need `as_deref_mut()`) or if we don't have a function call to cause it (e.g., a `match` on an enum of different context types).